feature-detects/css/hyphens.js

```diff
@@ -150,7 +150,14 @@ define(['Modernizr', 'prefixes', 'createElement', 'testAllProps', 'addTest'], fu
           document.body.insertBefore(div, firstChild);
           document.body.insertBefore(dummy, div);
 
-
+          var scrollX = (window.pageXOffset !== undefined)
+	          ? window.pageXOffset
```

Comment:
The build failed to due jshint bailing out on your ternary formatting, please fix.